src/index.ts

```diff
@@ -1,18 +1,33 @@
-const chainParams = require('./chains')
-const hardforkChanges = require('./hardforks')
+import { chains as chainParams } from './chains'
+import { hardforks as hardforkChanges } from './hardforks'
+
+interface hardforkOptions {
+  /** optional, only allow supported HFs (default: false) */
+  onlySupported?: boolean
+  /** optional, only active HFs (default: false) */
+  onlyActive?: boolean
+}
 
 /**
  * Common class to access chain and hardfork parameters
  */
-class Common {
+export class Common {
```

Comment:
Not sure if this is intended, but this changes how users import the class.

If the intention is to keep it as before, this should probably read `export default class Common`, and the tests should import it as `import Common from '../src/'`.

Reply:
Ah, still not so familiar with ES6 and TypeScript syntax, thanks for pointing this out.